docs/content/NavList.mdx

```diff
@@ -0,0 +1,523 @@
+---
+title: NavList
+status: Draft
+description: Use nav list to render a vertical list of navigation links.
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+To render a horizontal list of navigation links, consider using [UnderlineNav](/UnderlineNav).
+
+## Examples
+
+### Simple
+
+```jsx
+<NavList>
+  <NavList.Item href="/" aria-current="page">
+    Dashboard
+  </NavList.Item>
+  <NavList.Item href="/pulls">Pull requests</NavList.Item>
+  <NavList.Item href="/issues">Issues</NavList.Item>
+</NavList>
+```
+
+<details>
+  <summary>Rendered HTML</summary>
```

Comment:
Is the `<details>` and `<summary>` content meant to be inside the `html` codefence?

Reply:
No. These were a temporary way to get review on the rendered HTML without cluttering the document too much. I'll remove these `<details>` sections when I implement the component.